Mlem/App/Views/Pages/Person/PersonView.swift

```diff
@@ -96,8 +97,11 @@ struct PersonView: View {
                             savedOnly: false,
                             prefetchingConfiguration: .forPostSize(postSize)
                         )
-                        
                         preheatFeedLoader()
+                    } else if feedLoader?.api !== entity.api {
+                        Task {
+                            await feedLoader?.switchUser(api: entity.api, userId: entity.id)
+                        }
```

Comment:
*nit*

This prevents spinning up a `Task` if we don't have a `feedLoader`--the compiler might optimize that out, but it can't hurt

```suggestion
                    } else if let feedLoader, feedLoader.api !== entity.api {
                        Task {
                            await feedLoader.switchUser(api: entity.api, userId: entity.id)
                        }
```